README.md

```
@@ -1,2 +1,124 @@
+cd crabllet
+
+# Generate a 12-word mnemonic
+cargo run
+
+# Generate a 24-word mnemonic
+cargo run -- 24
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)